src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -70,7 +72,7 @@ static internal ExtraWorkspaceViewInfo ExtraWorkspaceViewInfoFromJson(string jso
                     Console.WriteLine(args.ErrorContext.Error);
                 },
                 ReferenceLoopHandling = ReferenceLoopHandling.Ignore,
-                TypeNameHandling = TypeNameHandling.Auto,
+                TypeNameHandling = TypeNameHandling.None,
```

Comment:
I see, you've made this change in general for all nodes. How will dyn deserialization work if the type information is missing?